thrifty-schema/src/main/kotlin/com/microsoft/thrifty/schema/Constant.kt

```diff
@@ -239,6 +231,21 @@ class Constant private constructor (
         }
     }
 
+    private object BinaryValidator : BaseValidator() {
+        override fun validate(symbolTable: SymbolTable, expected: ThriftType, valueElement: ConstValueElement) {
+            when (valueElement) {
```

Comment:
This `when` needs an `else -> super.validate(symbolTable, expected, valueElement)`.  Without it, we won't properly validate identifiers of binary constants.  We should have a test case for that, too:

```thrift
const binary FOO = "0xDEADBEEF"

struct S {
  1: required binary b = FOO;
}
```

Reply:
Totally forgot about this, I'll fix it during the holidays.